hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/om/TestOMDbCheckpointServlet.java

```diff
@@ -189,16 +197,19 @@ private void setupCluster() throws Exception {
         .thenReturn(cluster.getOzoneManager());
     when(requestMock.getParameter(OZONE_DB_CHECKPOINT_REQUEST_FLUSH))
         .thenReturn("true");
+    when(requestMock.getMethod()).thenReturn("POST");
+    when(requestMock.getContentType()).thenReturn("multipart/form-data; " +
+        "boundary=" + MULTIPART_FORM_DATA_BOUNDARY);
 
-    doCallRealMethod().when(omDbCheckpointServletMock).doGet(requestMock,
+    doCallRealMethod().when(omDbCheckpointServletMock).doPost(requestMock,
         responseMock);
 
     doCallRealMethod().when(omDbCheckpointServletMock)
         .writeDbDataToStream(any(), any(), any(), any(), any());
   }
 
   @Test
-  public void testDoGet() throws Exception {
+  public void testDoPost() throws Exception {
```

Comment:
Since we are keeping doGet() around for recon, we need to keep the doGet() test working as well.  The test is long and there is not much difference between the two versions, so I would prefer to parameterize this test so it runs twice, once with doGet() and once with doPost().  Is that difficult?